iml-gui/crate/src/lib.rs

```diff
@@ -29,7 +33,18 @@ const STATIC_PATH: &str = "static";
 const SLIDER_WIDTH_PX: u32 = 5;
 const MAX_SIDE_PERCENTAGE: f32 = 35f32;
 
-#[derive(Clone, Copy, Eq, PartialEq)]
+/// This depends on where and how https://github.com/whamcloud/Online-Help is deployed.
+/// With `nginx` when config is like
+/// ```
+/// location /help {
+///     alias /usr/lib/iml-manager/iml-online-help;
+///     index index.html;
+/// }
+/// ```
+/// help url becomes `https://localhost:8443/help/docs/Graphical_User_Interface_9_0.html`
+const CTX_HELP: &str = "help/docs/Graphical_User_Interface_9_0.html";
```

Comment:
Should the link be absolute?

Reply:
I think, it is too strong dependency, because the host and port number are pretty easy to change.
The downside is that we cannot have `Online-Help` on a different host, which can be fixed with an option in the (hypothetical) application config.

Reply:
I meant starting with a slash: `/help...` :)

Reply:
Hah, lol. I tried to do this, and this breaks the logic and refuses to open the new tab, instead it goes to 404 :-P
![image](https://user-images.githubusercontent.com/940363/70625195-1c882c80-1c4c-11ea-8be1-7ebef242b743.png)

Reply:
But removing `/` does not sound like a fix ;-) Something else is not right. What's going on? On your picture I don't see the full URL in the status bar before you click on Help.

Reply:
![image](https://user-images.githubusercontent.com/940363/70626075-ed72ba80-1c4d-11ea-9cf0-649164b91e8f.png)

Reply:
(More visual info)
![image](https://user-images.githubusercontent.com/940363/70626311-65d97b80-1c4e-11ea-8459-d1aae8111d99.png)

Reply:
Ok, it works because of `<base href="/">`

Reply:
You think `base href=""` is more preferrable in general case?

Reply:
I think it's best to leave this decision undefined for now. The existing GUI uses `/ui/` for it's base href (and is served from `/ui/` by nginx) which sidesteps the issue of `/help` being a different base.

We may / may not want to continue doing that, but I think we need to push this behind nginx in a production scenario before we decide what we should do here.